packages/jss/src/RuleList.js

```diff
@@ -200,16 +200,22 @@ export default class RuleList {
       sheet
     } = this.options
 
+    // updateFun is a workaround to use together
+    // jss-plugin-rule-value-function and jss-plugin-global
+    const hasUpdateFun = typeof rule.updateFun == 'function'
+
     // It is a rules container like for e.g. ConditionalRule.
-    if (rule.rules instanceof RuleList) {
+    // Ignore if there is an updateFun
+    if (!hasUpdateFun && rule.rules instanceof RuleList) {
```

Comment:
breaks encapsulation between plugins and the core, core has no knowledge about style functions, so far we managed to run the plugin in a way that doesn't require this

Reply:
the idea is that the plugin reacts to onUpdate hook and calls the function